docs/src/Icon.doc.js

```diff
@@ -18,6 +18,30 @@ Show icons with different colors and sizes in an accessible way.
   />
 );
 
+card(
+  <Box display="flex" direction="column">
+    <Text size="lg">
+      Icon component supports right-to-left(RTL) language locales layout (auto
+      flip on RTL locales like Arabic). Consider whether you want to keep the
+      original layout on rtl if you are adding new icons.
+    </Text>
+    <Box display="flex" direction="row">
```

Comment:
@selena-zeng Let's remove this `Box` since the rtl will be a feature across pages